### DISCUSSION TITLE
Issues and concerns with the scan change

### DISCUSSION THREAD

**Discussion by IsocratesGrey** [endless-sky/endless-sky#8520] [Balance, Mechanics, and Interactions]
Looks like there are several issues tagged regarding scanners, not sure if someone wants to merge these all into a single issue or project so that it can be tested in 0.10.1?

The move in 0.10.0 to turn scanning into a mini game makes a lot of sense but it has resulted in some bugs and unexpected issues in missions. Generally it looks like the concerns revolve around how it's impacted prior content such as: remnant ships love to scan so they hover around you now, missions that are designed for early ships now require bigger or faster ships, crime is too easy because the navy can't scan you.

Results from #6193

Relevant links so far, please add more:
#8493 #8492 #7640 #7876 #8426 #8447 #8395 #8338 #8323 #8272 #8288

**Comment by IsocratesGrey** [comment]
The most important discussion on this topic is probably this one, as it provides the context required:

https://github.com/endless-sky/endless-sky/issues/7640#issuecomment-1333766816

**Comment by IsocratesGrey** [comment]
My understanding is that scanning as mini-game is a balance of scanning power/efficiency, proximity to the ship being scanned, and length of time required to stay in that proximity.

If the mini-game is about proximity and time then the suggested solutions around increasing the number of scanners is basically a brute force method of avoiding the mini-game.

I suspect that a slight tweak to the proximity would solve this completely, either increasing the scan distance, or decreasing the penalty for scanning at max distance.

I also think that adjusting it so that adding more scanners doesn't linearly increase scan power would help too by making the brute force approach less useful. This could be done harshly by making only one scanner work at a time (the highest level for that type of scan), or more softly by providing greater diminishing returns for multiple scanners instead of a single high powered scanner.

**Reply by ziproot** [reply]
One PR suggests a "guassian dropoff," meaning that the rate at which scanning time increases is slow at first, then speeds up, then slows down again at the end. This means you still have to get close but not right on top anymore.
EDIT: Typo
